docs/release-notes.rst

```diff
@@ -147,6 +147,9 @@ Upgrade Notes
 
    * - Version
      - Notes
+   * - 25.4.0
+     - | - Whereabouts support will be deprecated in Network Operator 25.7.
```

Comment:
Isn't it deprecated now and will be removed in 25.7?

Reply:
Good question.
@e0ne do you know?

Are we going to remove from NCP CRD? It means have a new version of the CRD.
Maybe have it the CRD marked as `deprecated` only ?